skins/oasis/css/core/ads-porvata.scss

```diff
@@ -0,0 +1,282 @@
+		}
+	}
+}
+
+#INCONTENT_WRAPPER {
+	font-size: 0;
+	line-height: 0;
+	text-align: center;
+
+	#INCONTENT_PLAYER.porvata-outstream {
+		margin: 0 auto;
+		max-width: 640px;
+
+		&:before {
+			color: $color-text;
+			content: attr(data-label);
+			display: block;
+			font-size: 0.625rem;
+			font-weight: 700;
+			line-height: 0.75rem;
+			padding-bottom: 6px;
+			padding-top: 6px;
+			text-align: center;
+			text-transform: uppercase;
+		}
+
+		.video-overlay {
+			background: #000;
+		}
+	}
+}
+
+/* Outstream */
+
+.porvata-outstream {
+	.video-player {
+		.progress-bar {
+			background: $outstream-progress-bar-color;
+			height: 5px;
+		}
+
+		.current-time {
+			background: $outstream-progress-bar-current-time-color;
+		}
+
+		.ima-mute-div, .volume-control {
+			bottom: 14px;
+
+			.speaker {
+				&.mute {
+					background: #fff;
+					mask-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/outstream_sound_off_button.svg');
+					mask-position: center bottom;
+					mask-repeat: no-repeat;
+					-webkit-mask-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/outstream_sound_off_button.svg');
+					-webkit-mask-position: center bottom;
+					-webkit-mask-repeat: no-repeat;
+				}
+
+				background: #fff;
+				mask-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/outstream_sound_on_button.svg');
+				mask-position: center bottom;
+				mask-repeat: no-repeat;
+				-webkit-mask-image: url('/extensions/wikia/AdEngine/js/video/player/ui/images/outstream_sound_on_button.svg');
+				-webkit-mask-position: center bottom;
+				-webkit-mask-repeat: no-repeat;
+			}
+		}
+	}
+
+	.mobile-porvata .video-player > iframe {
+		// Hide Learn more text
+		z-index: -1;
+	}
+}
+
+/* Floating */
+
+#WikiaArticle.floating {
+	z-index: initial;
+
+	.wikia-ad {
+		bottom: $wikia-bar-height + $floating-ad-margin;
+		margin-bottom: 0;
+		position: fixed;
+		right: $floating-ad-margin;
+		transition: margin-bottom $floating-ad-animation-speed;
+		width: $floating-ad-video-width;
+		z-index: $floating-ad-index;
+
+		.provider-container {
+			padding-bottom: 0 !important; // has to be !important because o inline style
+			width: $floating-ad-video-width;
+		}
+
+		.floating-close-button {
+			display: block;
+		}
+	}
+
+	&.with-article-video {
+		.wikia-ad {
+			margin-bottom: $floating-ad-featured-video-gap;
+			transition: margin-bottom $floating-ad-animation-speed;
+		}
+	}
+}
+
+.WikiaTopAds.floating {
+	z-index: $floating-ad-index;
+
+	.wikia-ad {
+		bottom: $wikia-bar-height + $floating-ad-margin;
+		margin-bottom: 0;
+		position: fixed;
+		right: $floating-ad-margin;
+		transition: margin-bottom $floating-ad-animation-speed;
+
+		.video-player, iframe {
+			width: $floating-ad-video-width;
+		}
+
+		.floating-close-button {
+			display: block;
+		}
+	}
+
+	&.with-article-video {
+		.wikia-ad {
+			margin-bottom: $floating-ad-featured-video-gap;
+			transition: margin-bottom $floating-ad-animation-speed;
+		}
+	}
+}
```

Comment:
Can we have a common set of styles for `.floating` and add specific styles in separate blocks?

Reply:
Done